roledb.go

```diff
@@ -495,6 +495,34 @@ func (x *RoleGroupCache) InsertGroup(group *AuthGroup) (err error) {
 	return
 }
 
+func SaveGroup(icentral *Central, group *AuthGroup) bool {
```

Comment:
We don't need a SaveGroup function here, since it isn't adding any new functionality beyond what UpdateGroup already offers. Also, in Go one usually returns 'error' instead of a bool.